go/vt/vttablet/tabletmanager/vreplication/flags.go

```diff
@@ -39,6 +42,7 @@ var (
 	vreplicationHeartbeatUpdateInterval = 1
 	vreplicationExperimentalFlags       = int64(0x01) // enable vreplicationExperimentalFlagOptimizeInserts by default
 	vreplicationStoreCompressedGTID     = false
+	vreplicationParallelBulkInserts     = "auto"
```

Comment:
> I think that we should avoid using the experimental flag entirely and instead make the default be --vreplication_parallel_insert_workers=1 so that it's disabled by default.

Good suggestion, will do that